src/utils/proxy.js

```diff
@@ -52,6 +52,29 @@ const isExternal = function (match) {
   return match.to && match.to.match(/^https?:\/\//)
 }
 
+const stripOrigin = function (url) {
+  // Get the URL after http://host:port
+  return url.toString().replace(url.origin, '')
+}
+
+const proxyToExternalUrl = function ({ req, res, dest, destURL }) {
+  console.log(`${NETLIFYDEVLOG} Proxying to ${dest}`)
+  const handler = createProxyMiddleware({
+    target: `${dest.protocol}//${dest.host}`,
+    changeOrigin: true,
+    pathRewrite: () => destURL,
```

Comment:
The original code was stripping the origin using a regex:
https://github.com/netlify/cli/blob/a3b76061a32f67b24cba557524a91368df081a6d/src/utils/proxy.js#L200

This wasn't need since the origin is already stripped in:
https://github.com/netlify/cli/blob/a3b76061a32f67b24cba557524a91368df081a6d/src/utils/proxy.js#L184